examples/complete/main.tf

```diff
@@ -0,0 +1,99 @@
+## This is the module being used
+module "cis_alarms" {
+  # Use the git source in your own code
+  # source         = "git::https://github.com/cloudposse/terraform-aws-cloudtrail-cloudwatch-alarms.git?ref=<version>"
+  source = "../../"
+  log_group_name = aws_cloudwatch_log_group.default.name
+}
+
+## Everything after this is standard cloudtrail setup
+data "aws_caller_identity" "current" {}
+
+module "label" {
```

Comment:
For the example, you just need to add `context.tf` to `examples/complete/` and then use `module.this.id` and similar instead of creating a duplicate label here. See the review going on over [here](https://github.com/cloudposse/terraform-aws-named-subnets/pull/24#issuecomment-713633698).